plugin.yaml

```diff
@@ -1282,6 +1282,43 @@ relationships:
   cloudify.relationships.openstack.server_connected_to_port:
     derived_from: cloudify.relationships.connected_to
 
+  cloudify.relationships.openstack.server_connected_to_floating_ip:
```

Comment:
Should we add a message here that his is deprecated in the API?

Reply:
@EarthmanT I found the issue for this, it is solved not, the API is deprecated but we still can use it , the API is working it seems there is a little issue with calling the API